### DISCUSSION TITLE
Is "Embed on website" features will release (again)?

### DISCUSSION THREAD

**Discussion by joshua-666** [infiniflow/ragflow#3807] [Q&A]
like dify’s features:
<img width="254" alt="image" src="https://github.com/user-attachments/assets/e92ff57a-e344-4b4c-ac98-fd29963b1bf9">

![image](https://github.com/user-attachments/assets/5fdc32db-df14-4fb2-8acb-54534f8d1357)

Some features are available in version 0.12.

![image](https://github.com/user-attachments/assets/1180a676-d452-416c-b6a6-efb758ce70b0)

**Comment by KevinHuSh** [comment]
It will be.
